src/rewards/veterans.rs

```diff
@@ -249,23 +251,15 @@ mod tests {
     }
 
     #[test]
-    fn disagreement_discount_rate() {
+    fn disagreement_modifier_rate() {
         let total_rewards = Rewards::ONE;
         let inputs = [
-            (Rewards::new(4, 1), Rewards::ZERO, Rewards::ZERO),
-            (
-                Rewards::new(6, 1),
-                Rewards::from(2) / Rewards::from(3),
-                Rewards::ONE,
-            ),
-            (
-                Rewards::new(75, 2),
-                Rewards::from(5) / Rewards::from(6),
-                Rewards::ONE,
-            ),
-            (Rewards::new(9, 1), Rewards::ONE, Rewards::ONE),
+            (Rewards::new(6, 1), Rewards::ZERO, Rewards::ZERO),
+            (Rewards::new(7, 1), Rewards::new(75, 2), Rewards::ONE),
+            (Rewards::new(8, 1), Rewards::ONE, Rewards::ONE),
+            (Rewards::new(9, 1), Rewards::new(125, 2), Rewards::ONE),
         ];
```

Comment:
Unless I'm reading these numbers wrong, I feel like you may as well just write something like:

```rust
        let agreements = [
            Rewards::new(6, 1),
            Rewards::new(7, 1),
            Rewards::new(8, 1),
            Rewards::new(9, 1),
        ];

        let inputs = agreements.iter().copied().map(|agreement| {
            (
                agreement,
                rewards_disagreement_modifier(agreement),
                reputation_disagreement_modifier(agreement),
            )
        });

```

Reply:
Yes, it was a way to also check that `rewards_disagreement_modifier` and `reputation_disagreement_modifier` work as expected